src/SwedbankPay/Api/Client.php

```diff
@@ -0,0 +1,326 @@
+<?php
+// phpcs:ignoreFile
+
+namespace SwedbankPay\Api;
+
+use SwedbankPay\Api\Client\Exception;
+use SwedbankPay\Api\Response;
+
+/**
+ * Class Client
+ * @note For backward compatibility for previous versions
+ * @deprecated Use SwedbankPay\Api\Client\Client instead of
+ * @package SwedbankPay\Api
+ * @SuppressWarnings(PHPMD.CamelCasePropertyName)
+ */
+class Client
+{
+    const USER_AGENT = 'Swedbank Pay PHP SDK';
+    const MODE_PRODUCTION = 'production';
+    const MODE_TEST = 'test';
+
+    /**
+     * Curl Handler
+     *
+     * @var resource
+     */
+    protected $curl;
+
+    /**
+     * @var string|null
+     */
+    protected $last_response;
+
+    /**
+     * Merchant Token
+     *
+     * @var string
+     */
+    protected $merchant_token = '';
+
+    /**
+     * Test Mode
+     *
+     * @var string
+     */
+    protected $mode = 'test';
+
+    /**
+     * Api Endpoint
+     *
+     * @var string
+     */
+    protected $api_endpoint = 'https://api.externalintegration.payex.com';
+
+    /**
+     * Platform String
+     *
+     * @var string
+     */
+    protected $platform = '';
+
+    /**
+     * Client constructor.
+     * @SuppressWarnings(PHPMD.StaticAccess)
+     */
+    public function __construct()
+    {
+        $this->curl = curl_init();
+        curl_setopt_array($this->curl, [
+            CURLOPT_HTTP_VERSION   => CURL_HTTP_VERSION_1_1,
+            CURLOPT_RETURNTRANSFER => true,
+            CURLOPT_SSL_VERIFYPEER => true,
+            CURLOPT_SSL_VERIFYHOST => 2,
+            CURLOPT_CAINFO         => \SwedbankPay\Api\Client\Client::getSystemCaRootBundlePath(),
+            CURLOPT_TIMEOUT        => 60
+        ]);
+    }
+
+    /**
+     * Get Version
+     *
+     * @return string
+     */
+    public function getVersion()
+    {
+        $data = json_decode(file_get_contents(__DIR__ . '/../../../composer.json'), true);
```

Comment:
Would it be a good idea to write the version to a field in the class and return the class field if it's set so we don't need to read from the file system on every call to `getVersion()`?